tests/vm_test.sh

```diff
@@ -152,12 +152,12 @@ function test_vm_umount()
   assertEquals "($ID) - Expected 125" "$expected_ret" "$ret"
 
   ID=2
-  output=$(vm_umount "TEST_MODE" "" "$mount_point")
+  output=$(vm_umount "--umount" "TEST_MODE" "" "$mount_point")
```

Comment:
Since you are working on this part, could you also replace these `ID` variables with the `$LINENO` in the error message? This `ID` variable is a legacy thing that we want to remove.

Reply:
Also, use single quotes when you don't have a bash variable to be expanded.